cms/search/views.py

```diff
@@ -91,7 +92,7 @@ def get_date(before=True):
     query_string = request.GET.get("query", "")
     search_ordering = validated_sort_order(request.GET.get("order", None))
     search_type = request.GET.get("content_type", "")
-    publication_types = request.GET.getlist("publication_type")
+    publication_slugs = request.GET.getlist("publication_type")
```

Comment:
it might be worth being a bit more explicit with this variable, like `publication_type_slugs_to_include`.